app/app/room/[id].tsx

```
@@ -93,131 +95,299 @@
-          setChatReady(true);
+          // Begin chat initialization phase  
+          setInitPhase('chat');
+          // Start the chat phase timeout
+          watchPhaseTimeout('chat');
+          
+          console.log('[Room] Chat phase: Initializing chat data channel');
+          
+          // Initialize chat in the background
+          (async () => {
+            try {
+              chatManager.current = new ChatManager(newUserId, webrtcManager.current!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)